invenio_app_ils/ui/backoffice/src/pages/LoanDetails/components/LoanActions/tests/__snapshots__/LoanAction.test.js.snap

```diff
@@ -0,0 +1,284 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
I think it is going to be hard to review all the time the snapshots, especially with enzyme, they are not very human readable and quite long.
Here what our friend Ben from Eventbrite at the React workshop was saying:
https://github.com/benmvp/react-workshop/tree/master/src/testing/02-render-components

About snapshots:
`However, the ease of writing the test comes at a cost for maintaining the test. For one, the snapshot for the one test is over 100 lines. It's not very easy to read. But it's very easy for the test to fail when changing unrelated things.`
`Each one of these cause the test the fail, when all we really are trying to verify is that a corresponding <EmailListItem /> is created for every email in the emails prop array. Jest allows us to update the snapshot when it fails, but with multiple developers the snapshots will get auto-adopted without really verifying if the failure was real or just a false-positive.
Instead, manually verify the specific things we care about`

So basically, he was saying that with multiple people updating snapshots, it can be tricky, better checking the needed props manually:
`expect(emailListItem).toHaveProp('email', EMAILS[index]);`